language/nl/help_directory_flags.php

```diff
@@ -0,0 +1,267 @@
+	'AF' => 'Afghanistan',
+	'AG' => 'Antigua and Barbuda',
+	'AI' => 'Anguilla',
+	'AL' => 'Albanië',
+	'AM' => 'Armenië',
+	'AN' => 'Antillen',
+	'AO' => 'Angola',
+	'AQ' => 'Antarctica',
+	'AR' => 'Argentinië',
+	'AS' => 'Amerikaans Samoa',
+	'AT' => 'Oostenrijk',
+	'AU' => 'Australië',
+	'AW' => 'Aruba',
+	'AX' => 'Åland',
+	'AZ' => 'Azerbeidzjan',
+	'BA' => 'Bosnië-Herzegovina',
+	'BB' => 'Barbados',
+	'BD' => 'Bangladesh',
+	'BE' => 'België',
+	'BF' => 'Burkina Faso',
+	'BG' => 'Bulgarije',
+	'BH' => 'Bahrein',
+	'BI' => 'Burundi',
+	'BJ' => 'Benin',
+	'BL' => 'St. Barthelemy',
+	'BM' => 'Bermuda',
+	'BN' => 'Brunei',
+	'BO' => 'Bolivia',
+	'BR' => 'Brazilië',
+	'BS' => 'Bahama‘s',
+	'BT' => 'Bhutan',
+	'BV' => 'Bouveteiland',
+	'BW' => 'Botswana',
+	'BY' => 'Wit-Rusland',
+	'BZ' => 'Belize',
+	'CA' => 'Canada',
+	'CC' => 'Cocoseilanden',
+	'CD' => 'Congo, de Democratische Republiek',
+	'CF' => 'Centraal-Afrikaanse Republiek',
+	'CG' => 'Congo',
+	'CH' => 'Zwitserland',
+	'CI' => 'Ivoorkust',
+	'CK' => 'Cook eilanden',
+	'CL' => 'Chili',
+	'CM' => 'Kameroen',
+	'CN' => 'China',
+	'CO' => 'Colombia',
+	'CR' => 'Costa Rica',
+	'CU' => 'Cuba',
+	'CV' => 'Kaapverdië',
+	'CX' => 'Christmas eiland',
+	'CY' => 'Cyprus',
+	'CZ' => 'Tsjechië',
+	'DE' => 'Duitsland',
+	'DJ' => 'Djibouti',
+	'DK' => 'Denemarken',
+	'DM' => 'Dominica',
+	'DO' => 'Dominicaanse Republiek',
+	'DZ' => 'Algerije',
+	'EC' => 'Ecuador',
+	'EE' => 'Estland',
+	'EG' => 'Egypte',
+	'EH' => 'Westelijke Sahara',
+	'ER' => 'Eritrea',
+	'ES' => 'Spanje',
+	'ET' => 'Ethiopië',
+	'FI' => 'Finland',
+	'FJ' => 'Fiji',
+	'FK' => 'Falklandeilanden',
+	'FM' => 'Micronesia',
+	'FO' => 'Faeröer',
+	'FR' => 'Frankrijk',
+	'GA' => 'Gabon',
+	'GB' => 'Verenigd Koninkrijk',
+	'GD' => 'Grenada',
+	'GE' => 'Georgië',
+	'GF' => 'Frans-Guyana',
+	'GG' => 'Guernsey',
+	'GH' => 'Ghana',
+	'GI' => 'Gibraltar',
+	'GL' => 'Groenland',
+	'GM' => 'Gambia',
+	'GN' => 'Guinea',
+	'GP' => 'Guadeloupe',
+	'GQ' => 'Equatoriaal-Guinea',
+	'GR' => 'Griekenland',
+	'GS' => 'Zuid-Georgië en de Zuidelijke Sandwich-eilanden',
+	'GT' => 'Guatemala',
+	'GU' => 'Guam',
+	'GW' => 'Guinee-Bissau',
+	'GY' => 'Guyana',
+	'HK' => 'Hong Kong',
+	'HM' => 'Heard en McDonald Eilanden',
+	'HN' => 'Honduras',
+	'HR' => 'Kroatië',
+	'HT' => 'Haïti',
+	'HU' => 'Hongarije',
+	'ID' => 'Indonesië',
+	'IE' => 'Ierland',
+	'IL' => 'Israël',
+	'IM' => 'Man (eiland)',
+	'IN' => 'Indië',
+	'IO' => 'Britse Territorium in de Indische Oceaan',
+	'IQ' => 'Irak',
+	'IR' => 'Iran',
+	'IS' => 'IJsland',
+	'IT' => 'Italië',
+	'JE' => 'Jersey',
+	'JM' => 'Jamaica',
+	'JO' => 'Jordanië',
+	'JP' => 'Japan',
+	'KE' => 'Kenia',
+	'KG' => 'Kirghizistan',
+	'KH' => 'Cambodja',
+	'KI' => 'Kiribati',
+	'KM' => 'Comores',
+	'KN' => 'Saint Kitts and Nevis',
+	'KP' => 'Noord Korea',
+	'KR' => 'Zuid Korea',
+	'KW' => 'Koeweit',
+	'KY' => 'Kaaimaneilanden',
+	'KZ' => 'Kazachstan',
+	'LA' => 'Laos',
+	'LB' => 'Liban', //????
```

Comment:
I guess you're right, it looks like French it should be English ;) So you might want to check the English variable for this...